### PR TITLE
adding ci configuration

### LINKED ISSUE #57: integrate karma tests to the CI of this repo

### PR DESCRIPTION
**Pull request by caridy**
closes #57

### CODE REVIEW

**Comment by caridy** [comment]
@manuel-jasso this is the ci config... which is closing #57, there is obviously one test failing... the one async await, not sure what to do about that at the moment, nevertheless, merging this as it is once reviewed.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - run: yarn install
+    - run: yarn build
+    - run: yarn test
+    - run: yarn karma
+      env:
+        CI: true
```
Comment (manuel-jasso) [inline .github/workflows/main.yml:31]: new line nit

**Comment by manuel-jasso** [comment]
> @manuel-jasso this is the ci config... which is closing #57, there is obviously one test failing... the one async await, not sure what to do about that at the moment, nevertheless, merging this as it is once reviewed.

Let me disable that test for Firefox in the mean time.

**Comment by manuel-jasso** [comment]
> > @manuel-jasso this is the ci config... which is closing #57, there is obviously one test failing... the one async await, not sure what to do about that at the moment, nevertheless, merging this as it is once reviewed.
> 
> Let me disable that test for Firefox in the mean time.

See #116

File: .github/workflows/main.yml (hunk 1 above)
Comment (caridy) [inline .github/workflows/main.yml:31]:
```suggestion
        CI: true

```

**Pull request merged**